### ISSUE TITLE
Location Controls - hard to understand.

### ISSUE THREAD

**Issue by paaljoachim** [joedolson/my-calendar#159]
I noticed a section above the promotional sidebar called Location Controls (sounds like a setting that belongs in the settings screen to me..:)

![Screen Shot 2021-07-27 at 21 41 48](https://user-images.githubusercontent.com/5323259/127217567-128584d7-f303-4b4b-9a00-0d0c9d76e7ac.jpg)

Let's say I wanted to remove Region from the Add New Location. I would open the Region see the default values of "saved_value,Displayed Value" and carefully back away. Not knowing if I should touch anything there. It just seems too complex and not needed as part of the Add New Location screen. 

If it is needed then it could perhaps instead be added to the Input section inside Settings.

**Comment by joedolson** [comment]
Definitely can't be removed; some users make heavy use of this. But yes, it's confusing to set up. It allows you to control the data allowed in the fields, e.g. if you only supported events in the United States and Canada, you could restrict the content to those fields, and the field would convert to a select input that only allowed those two values.

**Comment by paaljoachim** [comment]
Ok, so this is a section we will likely need to add a link to documentation, so users can learn what it does.

**Comment by joedolson** [comment]
Yes. Once the documentation is posted publicly, that will be a good option for a variety of more complicated things.

**Comment by paaljoachim** [comment]
I will gradually begin adding Documentation issues to the repo and add the needed information from the current My Calendar pdf to each issue.
